addons/web/static/tests/webclient/actions/report_action_tests.js

```diff
@@ -354,4 +354,35 @@ QUnit.module("ActionManager", (hooks) => {
         assert.verifySteps(["/report/check_wkhtmltopdf", "/report/download"]);
         testUtils.mock.unpatch(ReportClientAction);
     });
+
+    QUnit.test("url is valid", async (assert) => {
+        assert.expect(2);
+
+        patchWithCleanup(ReportClientAction.prototype, {
+            init() {
+                this._super(...arguments);
+                this.report_url = "about:blank";
+            },
+        });
+
+        const webClient = await createWebClient({ serverData });
+
+        const action = {
+            context: {
+                rabbia: "E Tarantella",
+                active_ids: [99],
+            },
+            data: null,
+            name: "Ennio Morricone",
+            report_name: "ennio.morricone",
+            report_type: "qweb-html",
+            type: "ir.actions.report",
+        };
```

Comment:
suggestion to have a more pertinent scenario:
give an id to the action and add the action to the list of actions
do `doAction(webClient, _id_)`
check that the url contains "action=`_id_`"